modules/local/metafusion/run/main.nf

```diff
@@ -37,12 +38,13 @@ process METAFUSION_RUN {
         --gene_info $info \\
         --genome_fasta $fasta \\
         --recurrent_bedpe $blocklist \\
+        --clinical_genes $clinicalgenes \\
         ${args}
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
         Metafusion docker: \$METAFUSION_TAG
```

Comment:
although this wasn't added in this PR please remove the word " docker" -- every image is available in docker.